packages/backend/src/graphql/schema.graphql

```diff
@@ -79,6 +79,8 @@ type Mutation {
   createRows(input: CreateTableRowsInput!): Boolean
   updateRow(input: UpdateTableRowInput!): ID!
   deleteRows(input: DeleteTableRowsInput!): [ID!]!
+  # Flow transfers
+  createFlowTransfer(input: CreateFlowTransferInput!): ID! # TODO (mal): check if need return transfer
```

Comment:
can go ahead and return the entire object to make changing stuff simpler later on

Reply:
ok got it, thanks!